configs/test-utils/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@suspensive/test-utils",
+  "version": "0.0.0",
+  "private": true,
+  "funding": {
+    "type": "github",
+    "url": "https://github.com/sponsors/manudeli"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Jonghyeon Ko",
+    "email": "manudeli.ko@gmail.com"
+  },
+  "sideEffects": false,
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "dist/index.cjs",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsup",
+    "build:watch": "tsup --watch",
```

Comment:
Is this build config required? I think there is no need to build this files and need to just export pure ts file. isn't it?

Reply:
Actually, I tried it before, but it didn't work properly so I gave up. However, when I looked again, I should have used index.tsx as main, but the problem occurred because I used index.ts.

So, to avoid confusion in the first place, we unified them into index.ts.
Thank you for giving me motivation to try again